packages/contracts/contracts/verifierHelpers/VerifierHelperManager.sol

```diff
@@ -3,8 +3,10 @@ pragma solidity ^0.8.0;
 
 import { IVerifierHelper } from "../interfaces/IVerifierHelper.sol";
 import { BaseVerifierHelper } from "@unirep/contracts/verifierHelpers/BaseVerifierHelper.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
 
-contract VerifierHelperManager {
+contract VerifierHelperManager is Ownable {
+    address public app;
```

Comment:
what is the purpose of this app? If this does not have any functionality, then it should be removed :)

Reply:
sorry, forgot to remove~